app/src/main/java/com/github/ybecker/epforuml/latex/MathView.kt

```diff
@@ -0,0 +1,187 @@
+        <script type="text/javascript" src="file:///android_asset/katex/katex.min.js" ></script>
+        <script type="text/javascript" src="file:///android_asset/katex/contrib/auto-render.min.js" ></script>
+        <script type="text/javascript" src="file:///android_asset/katex/contrib/auto-render.js" ></script>
+        <script type="text/javascript" src="file:///android_asset/jquery.min.js" ></script>
+        <script type="text/javascript" src="file:///android_asset/latex_parser.js" ></script>
+        <meta name="viewport" content="width=device-width"/>
+<link rel="stylesheet" href="file:///android_asset/webviewstyle.css"/>
+<style type='text/css'>body {margin: 0px;padding: 0px;font-size:${text_size}px;color:${
+                getHexColor(
+                    text_color
+                )
+            }; } </style>    </head>
+    <p style="word-wrap: break-word;">
+        {formula}
+    </p>
+</html>"""
+            val start =
+                "<html><head><meta http-equiv='Content-Type' content='text/html' charset='UTF-8' /><style> body {" +
+                        " white-space: nowrap;}</style></head><body>"
+            val end = "</body></html>"
+            return offline_config.replace("{formula}", display_text!!)
+        }
+
+    fun setTextSize(size: Int) {
+        text_size = size
+        loadData()
+    }
+
+    fun setTextColor(color: Int) {
+        text_color = color
+        loadData()
+    }
+
+    private fun getHexColor(intColor: Int): String {
+        //Android and javascript color format differ javascript support Hex color, so the android color which user sets is converted to hexcolor to replicate the same in javascript.
+        val hexColor = String.format("#%06X", 0xFFFFFF and intColor)
+        Log.d(TAG, "Hex Color:$hexColor")
+        return hexColor
+    }
+
+    private fun setDefaultTextColor(context: Context) {
+        //sets default text color to black
+        text_color = ContextCompat.getColor(context, R.color.black)
+    }
+
+    private fun setDefaultTextSize() {
+        //sets view default text size to 18
+        text_size = default_text_size.toInt()
+    }
+
+    private fun loadData() {
+        if (display_text != null) {
+            loadDataWithBaseURL("null", offlineKatexConfig, "text/html", "UTF-8", "about:blank")
+        }
+    }
+
+    override fun setClickable(is_clickable: Boolean) {
+        this.isEnabled = true
+        clickable = is_clickable
+        enable_zoom_in_controls = !is_clickable
+        configurationSettingWebView(enable_zoom_in_controls)
+        this.invalidate()
+    }
+
+    @SuppressLint("NewApi")
+    override fun onTouchEvent(event: MotionEvent): Boolean {
+        return if (clickable && event.action == MotionEvent.ACTION_DOWN) {
+            callOnClick()
+            false
+        } else {
+            super.onTouchEvent(event)
+        }
+    }
+
+    companion object {
```

Comment:
constants should be defined in capital letters and the use of _ is not really recommended